ambari-server/src/main/java/org/apache/ambari/server/controller/DeleteIdentityHandler.java

```diff
@@ -281,6 +327,12 @@ private String dataDirectory() {
     private KerberosDescriptor getKerberosDescriptor() throws AmbariException {
       return getKerberosHelper().getKerberosDescriptor(getCluster(), false);
     }
+
+    @Override
```

Comment:
I see the same method below starting in line 376; wondering why there is no any compilation error...

Reply:
They are in different nested classes:
* PrepareDeleteIdentityServerAction.pruneServiceFilter
* DeleteDataDirAction.pruneServiceFilter

Reply:
Ahh...GitHub hid the common parts...and I did not expanded those sections